packages/guides/integrating-the-safe-core-sdk.md

```diff
@@ -0,0 +1,353 @@
+
+const safeAccountConfig: SafeAccountConfig = {
+  owners: ['0x...', '0x...', '0x...']
+  threshold: 2,
+  // ... (optional params)
+}
+const safeSdk = await safeFactory.deploySafe(safeAccountConfig)
+```
+
+Calling the method `deploySafe` will deploy de desired Safe and return a Safe Core SDK initialized instance ready to be used.
+
+## 4. Create a transaction <a name="create-transaction"></a>
+
+The Safe Core SDK supports the execution of single Safe transactions but also MultiSend transactions. We can create a transaction object by calling the method `createTransaction` in our `Safe` instance.
+
+* **Create a single transaction**
+
+  This method can receive an object of type `SafeTransactionDataPartial` that represents the transaction we want to execute (once the signatures are collected). There are some optional properties in case we want to specify them.
+
+  ```js
+  import { SafeTransactionDataPartial } from '@gnosis.pm/safe-core-sdk-types'
+
+  const transaction: SafeTransactionDataPartial = {
+    to,
+    data,
+    value,
+    operation, // Optional
+    safeTxGas, // Optional
+    baseGas, // Optional
+    gasPrice, // Optional
+    gasToken, // Optional
+    refundReceiver, // Optional
+    nonce // Optional
+  }
+
+  const safeTransaction = await safeSdk.createTransaction(transaction)
+  ```
+
+* **Create a MultiSend transaction**
+
+  This method can receive an array of `MetaTransactionData` objects that represent the multiple transactions we want to include in our MultiSend transaction. If we want to specify some of the optional properties in our MultiSend transaction we can pass a second argument to the method `createTransaction` with the `SafeTransactionOptionalProps` object.
+
+  ```js
+  import { SafeTransactionOptionalProps } from '@gnosis.pm/safe-core-sdk'
+  import { MetaTransactionData } from '@gnosis.pm/safe-core-sdk-types'
+
+  const transactions: MetaTransactionData[] = [
+    {
+      to,
+      data,
+      value,
+      operation
+    },
+    {
+      to,
+      data,
+      value,
+      operation
+    },
+    // ...
+  ]
+
+  const options: SafeTransactionOptionalProps = {
+    safeTxGas, // Optional
+    baseGas, // Optional
+    gasPrice, // Optional
+    gasToken, // Optional
+    refundReceiver, // Optional
+    nonce // Optional
+  }
+
+  const safeTransaction = await safeSdk.createTransaction(transactions, options)
+  ```
+
+
+We can specify the `nonce` of our Safe transaction as long as it is not lower than the current Safe nonce. If multiple transactions are created but not executed they will share the same `nonce` if no `nonce` is specified, making valid the first transaction that gets executed and invalidating all the rest. We can prevent this by calling the method `getNextNonce` from the Safe Service Client instance. This method will take into account all the transactions that are queued and pending to be executed and calculate the next nonce, making it different for all the transactions.
```

Comment:
"...making valid the first transaction that gets executed and invalidating all the rest." => "...validating the first executable transaction and invalidating the rest."

Reply:
"This method takes all queued/pending transactions into account when calculating the next nonce, creating a unique one for all different transactions"

Reply:
I'm not sure your first comment here means the same.
All are valid (and executable) until one is executed. Once any of them is executed, the rest will become invalid.

Reply:
"...validating the first executed transaction and invalidating the rest."